suite/tests/client-interface/drreg-test-shared.h

```diff
@@ -81,6 +81,7 @@
 #endif
 
 #define TEST_CONST MAKE_HEX_ASM(0)
+#define TEST_CONST_2 MAKE_HEX_ASM(12)
```

Comment:
nit: maybe a better name, e.g. TEST_11_CONST and TEST_12_CONST, or TEST_0_CONST and TEST_12_CONST?

Reply:
Done